docs/spec/v2/helmreleases.md

```diff
@@ -538,6 +540,8 @@ The field offers the following subfields:
   rendered templates against the Kubernetes OpenAPI Schema. Defaults to `false`.
 - `.disableSchemaValidation` (Optional): Prevents Helm from validating the
   values against the JSON Schema. Defaults to `false`.
+- `.disableTakeOwnership` (Optional): Disables ignoring the check for helm labels and annotations before taking
+  ownership of the existing resources during the Helm install action. Defaults to `false`.
```

Comment:
```suggestion
  ownership of the existing resources during the Helm upgrade action. Defaults to `false`.
```

Reply:
My bad!

Reply:
Did you really fix it? I'm still seeing `install` here.

Reply:
My bad again 🤦 . Incorporated the reviews. Please take a look again, thanks!